.prettierignore

```diff
@@ -0,0 +1,17 @@
+# Add files here to ignore them from prettier formatting
+
+/**/__generated__
+/**/dist
```

Comment:
These 2 lines are probably not needed, think this was a left-over from the old Nx setup we had before